testing/hamlib/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Contributor: Bradford D. Boyle <bradford.d.boyle@gmail.com>
+# Maintainer: Bradford D. Boyle <bradford.d.boyle@gmail.com>
+pkgname=hamlib
+pkgver=3.0
+pkgrel=0
+pkgdesc="Library to control radio transceivers and receivers"
+url="http://www.hamlib.org/"
+arch="all"
+license="LGPLv2+"
+depends="perl python"
+depends_dev=""
+makedepends="$depends_dev linux-headers perl-dev python-dev swig tcl-dev"
+install=""
+subpackages="$pkgname-dev $pkgname-doc"
+source="http://downloads.sourceforge.net/project/hamlib/hamlib/3.0/hamlib-3.0.tar.gz"
+
+builddir="$srcdir"/$pkgname-$pkgver
+build() {
+        cd "$builddir"
+        ./configure --prefix=/usr --sbindir=/usr/bin \
+                --with-perl-binding \
+                --with-python-binding \
+                --with-tcl-binding
+        make
+}
+
+package() {
+        cd "$builddir"
+        make DESTDIR="$pkgdir" install
+
+        install -Dm644 COPYING "$pkgdir"/usr/share/licenses/$pkgname/COPYING
+
+        # fix perl module location
+        cd "$pkgdir"/usr/lib/perl5/site_perl/
+        mkdir current
+        mv auto current
+        mv Hamlib.pm current
+        rm perltest.pl
+
+        /usr/bin/find "$pkgdir" -name '.packlist' -delete
+        /usr/bin/find "$pkgdir" -name '*.pod' -delete
```

Comment:
I was copying the [PKGBUILD from Arch](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=hamlib), but now that I've split the package into subpackages, I've left them in with the perl subpackage.